### PR TITLE
docs: change order in readme

### PR DESCRIPTION
**Pull request by sivangbagri**
Solving issue #291

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,8 +39,15 @@ correspondences between the packages may be summarized as:
...
 
+# Pre-requisite
+ [Python](https://www.python.org/) | [3.10](https://www.python.org/downloads/release/python-3100/) 
+
 # Installation
 
+## Creating a Virtual Environment
+
+    python3.10 -m venv venv/3.10
+    source venv/3.10/bin/activate
```
Comment (korikuzma) [inline README.md:50]: We'd probably want to move the installing for development section here.

**Review by korikuzma — CHANGES_REQUESTED** [review]

**Comment by larrybabb** [comment]
@korikuzma can you verify this is what you expected. I'll do the merge if you approve it.

**Comment by korikuzma** [comment]
@sivangbagri Can you pull in the latest changes from main and update your branch?

File: README.md [hunk 2]
```diff
@@ -28,17 +28,64 @@ The ga4gh/vrs-python repo embeds the ga4gh/vrs repo as a git submodule, and
...
+| [0.7](https://github.com/ga4gh/vrs-python/tree/0.7)   | [1.2](https://github.com/ga4gh/vrs/tree/1.2)                             |
+| [0.8](https://github.com/ga4gh/vrs-python/tree/0.8)   | [1.3](https://github.com/ga4gh/vrs/tree/1.3)                             |
+| [0.9](https://github.com/ga4gh/vrs-python/tree/0.9)   | [metaschema-update](https://github.com/ga4gh/vrs/tree/metaschema-update) |
 
 ⚠ **Developers: See the development section below for recommendations for using submodules
 gracefully (and without causing problems for others!).**
 
+# Pre-requisite
+
+[Python](https://www.python.org/) | [3.10](https://www.python.org/downloads/release/python-3100/)
```
Comment (korikuzma) [inline README.md:44]:
```suggestion
[Python 3.10](https://www.python.org/downloads/release/python-3100/)
```

File: README.md [hunk 3]
```diff
@@ -104,7 +151,6 @@ Here are some things to try.
 
 Once installed as described above, type
 
-    $ source venv/3.10/bin/activate
```
Comment (korikuzma) [inline README.md:107]: You can add this back and just make a comment to activate the venv if it's not already

**Review by korikuzma — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -87,7 +87,10 @@ then set the `VRS_SCHEMA_DIR` environment variable to use an alternative schema
 location.
 
 # Installation
-
+Activate the virtual environment (if not already) :
+  
+    $ source venv/3.10/bin/activate
+
```
Comment (korikuzma) [inline README.md:93]: Move this after lines 60-64

```markdown

Fork the repo at https://github.com/ga4gh/vrs-python/ .

    $ git clone --recurse-submodules git@github.com:YOUR_GITHUB_ID/vrs-python.git
    $ cd vrs-python
    $ make devready

Activate the virtual environment:

    $ source venv/3.10/bin/activate
```

File: README.md [hunk 5]
```diff
@@ -149,6 +152,8 @@ Here are some things to try.
 
 # Running the Notebooks
 
+
+
```
Comment (korikuzma) [inline README.md:156]: Remove new lines

**Review by korikuzma — CHANGES_REQUESTED** [review]

**Review by larrybabb — APPROVED** [review]
+1

**Pull request merged**